### DISCUSSION TITLE
tutorials/lift-off-part5/04-deploying-apollo-server

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#95] [course feedback]
# tutorials/lift-off-part5/04-deploying-apollo-server

https://www.apollographql.com/tutorials/lift-off-part5/04-deploying-apollo-server

**Comment by NnaDid** [comment]
I have developed an applicatiion with REACT, apollo client and apollo server, how can I deploy  both backend and frontend to a shared hosting service?

**Reply by mabuyo** [reply]
Hey there @NnaDid - this course is deprecated so we don't maintain it actively. I think the instructions should still be relevant for your question though.

**Reply by NnaDid** [reply]
Thank you very much @mabuyo but is there any active course now  that I may follow?

**Reply by mabuyo** [reply]
We don't, unfortunately but the docs for whichever hosting service you're using should be able to help you!

**Reply by NnaDid** [reply]
okay thak you very much @mabuyo
